Assets/SEE/Utils/SEEMath.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+
+namespace SEE.Utils
+{
+    /// <summary>
+    /// This class contains additional mathematical functions.
+    /// <para>
+    /// Individual methods may be copied from other sources like newer versions of dotnet framework.
+    /// Please consult method descriptions for credits and license.
+    /// </para>
+    /// </summary>
+    public static partial class SEEMath
```

Comment:
Why is this class `partial`?

Reply:
I initially intended only to include content with the same license and origin in the file and create a different one for our own code.